bitvm/src/chunk/wrap_hasher.rs

```diff
@@ -4,7 +4,6 @@ use crate::{
     signatures::wots_api,
```

Comment:
Please run `clippy --all-targets` and fix the warnings in `blake3.rs`. #259 should fix the warnings in all other files.